reporters/kamon-datadog/src/main/resources/reference.conf

```diff
@@ -62,6 +62,9 @@ kamon {
       connect-timeout = 5 seconds
       read-timeout = 5 seconds
       write-timeout = 5 seconds
+
+      # Use 'Deflate' compression when posting to the Datadog API
+      compression = false
```

Comment:
you might also have to update their docs for the Datadog exporter. they should be in the same repo